src/pages/products/components/FilterSection.jsx

```diff
@@ -38,7 +73,26 @@ function FilterSection() {
 							id="checkbox-fungames"
 							type="checkbox"
 							name="checkbox"
-							value="fun games"
+							value="FUN_GAMES"
+							onChange={(e) => {
```

Comment:
you can write onchange handler in one function

Reply:
Got it, thank you! Made the change in my next commit